core/src/com/mygdx/crazybomber/MainRooms.java

```diff
@@ -0,0 +1,136 @@
+//TODO: right now this is basically just MainMaps and goes to the map class,
+// Need to make it go to a room instead, which is the room class
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+public class MainRooms implements Screen {
+    private Stage stage;
+    private Table table;
+    private TextButton buttonRoom1,buttonRoom2,buttonRoom3,buttonBack;
+    private Label heading;
+    private Skin skin;
+    private TextureAtlas atlas;
+    private Sprite background;
+    private SpriteBatch batch;
+    private MapRepository mapRepository;
+    private RoomRepository roomRepository;
+    private StateRepository stateRepository;
+
+    public MainRooms(MapRepository mapRepository, RoomRepository roomRepository, StateRepository stateRepository) {
+        this.mapRepository = mapRepository;
+        this.roomRepository = roomRepository;
+        this.stateRepository = stateRepository;
+    }
+    @Override
+    public void show() {
+        batch = new SpriteBatch();
+        background = new Sprite(new Texture("background1.png"));
+        background.setSize(Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        stage = new Stage();
+        Gdx.input.setInputProcessor(stage);
+        atlas = new TextureAtlas("ui/button.pack");
+        skin = new Skin(Gdx.files.internal("menuSkin.json"),atlas);
+        table = new Table(skin);
+        table.setBounds(0,0,Gdx.graphics.getWidth(),Gdx.graphics.getHeight());
+
+        buttonRoom1 = new TextButton("ROOM1",skin);
```

Comment:
this should be done in a foreach using the roomRepository. make the textButtons into an list  such that the size is dynamic